spec/factories/cloud_object_store.rb

```diff
@@ -0,0 +1,17 @@
+FactoryGirl.define do
+  factory :aws_object, :class => ManageIQ::Providers::Amazon::StorageManager::S3::CloudObjectStoreObject.name do |object|
+    object.sequence(:key) { |n| "object-key-#{n}" }
+  end
+
+  factory :aws_bucket_with_objects,
+          :class => ManageIQ::Providers::Amazon::StorageManager::S3::CloudObjectStoreContainer.name do |bucket|
+    bucket.sequence(:name) { |n| "stubbed-name-#{n}" }
+    bucket.sequence(:ems_ref) { |n| "stubbed-ems-ref-#{n}" }
+
+    after(:create) do |b|
```

Comment:
Rubocop pointed his gun to me saying "Shadowing outer local variable - bucket". But I too prefer explicit names so fuck rubocop, I'm using `bucket` here. Thanks for pointing this out.